src/pages/signup/eu-cloud.js

```diff
@@ -61,37 +58,42 @@ export default function SelfHost() {
                                 <Plan
                                     className="px-0"
                                     title="While you're waiting..."
-                                    subtitle="We'll get back to you when PostHog Cloud EU is ready, but you can get started today on PostHog Cloud (US), or you can self-hostm, then migrate your event data when we're ready for you."
+                                    subtitle="We'll get back to you when PostHog Cloud EU is ready. In the mean time, you can use PostHog Cloud (US), or you can self-host, then migrate data when we're ready for you."
```

Comment:
I believe _meantime_ it one word

Reply:
This is a thing of beauty!

Reply:
![](http://jeremyvarner.com/blog/wp-content/uploads/2016/08/correct.gif)


```suggestion
                                    subtitle="We'll get back to you when PostHog Cloud EU is ready. In the meantime, you can use PostHog Cloud (US), or you can self-host, then migrate data when we're ready for you."
```

Reply:
https://github.com/PostHog/posthog.com/pull/4024/